CORRELATION_CONTEXT_HTTP_HEADER_FORMAT.md

```diff
@@ -0,0 +1,64 @@
+# Correlation Context HTTP Header Format
+
+A correlation context header is used to pass the name-value context properties (also known as tags) for the trace. This is a companion header for the `Trace-Context`. The values are intended for control of logging systems and should be passed along to any child requests. 
+
+# Format
+
+## Header name
+
+`Correlation-Context`
+
+Multiple correlation context headers are allowed. Values can be combined in a single header according to [rfc](https://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html#sec4.2).
+
+## Field value
+
+`name1=value1[;type=(number|bool|string)], name2=value2[;type=(number|bool|string)]`
+
+Overall Correlation-Context length MUST NOT exceed `1024` bytes. Key and value length should stay well under the combined limit of `1024` bytes. 
```

Comment:
Does the combined limit apply to a single header or across all correlation headers combined? I'd prefer just one.

Reply:
If it is just one can we have a lower limit for one? Let's say something like:
* keys max size X bytes
* string values max size X bytes
* numbers have to be also size limited if we use the current format definition.
* Maybe a total max headers?

Reply:
proposed new limits. The idea of proposed limits: 
- url should fit easily as a value
- overall size should not be driven by max number multiply by max size as it will become too big
- max number of properties matches max number of cookies typically supported by browser for a single domain (generally browsers supports less, just took Chrome's number for a reference)